src/routing/signOutRouteGuard.ts

```diff
@@ -34,9 +34,13 @@ export class SignOutRouteGuard implements RouteGuard {
                         const identity = await this.mapiClient.get<Identity>("/identity", [await this.mapiClient.getPortalHeader("delegationSignOut")]);
```

Comment:
The `usersService` has `getCurrentUserId()` method. You could re-use it here.

Reply:
If we use UserService we will lost info about delegation user signin.